### PR TITLE
CPLP-786 Connector API groundwork: database extensions

### PR DESCRIPTION
**Pull request by wira-mms**

### CODE REVIEW

File: src/portalbackend/CatenaX.NetworkServices.PortalBackend.PortalEntities/Entities/Connector.cs [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+public class Connector
+{
+    public Connector(string name, string locationId, string connectorUrl)
+    {
+        Name = name;
+        LocationId = locationId;
+        ConnectorUrl = connectorUrl;
+    }
+
+    public Guid Id { get; set; }
```
Comment (ntruchsess) [inline src/portalbackend/CatenaX.NetworkServices.PortalBackend.PortalEntities/Entities/Connector.cs:15]: the setter of the key-field should be private

**Comment by ntruchsess** [comment]
For better overview there should be single new Migration per PR, nobody will want to find out which intermediate commit this second Migration is referring to.

**Review by ntruchsess — CHANGES_REQUESTED** [review]

File: src/portalbackend/CatenaX.NetworkServices.PortalBackend.PortalEntities/Entities/Connector.cs (hunk 1 above)
Comment (wira-mms) [reply src/portalbackend/CatenaX.NetworkServices.PortalBackend.PortalEntities/Entities/Connector.cs]: https://github.com/catenax-ng/product-portal-backend/pull/60/commits/e2abce43c323d399b9d7e6b9fea4afb29c4924af

**Comment by wira-mms** [comment]
> For better overview there should be a single new Migration per PR, nobody will want to find out which intermediate commit this second Migration is referring to. Just do 'dotnet ef migrations remove' two times followed by a single 'dotnet ef migrations add xxx'

Tagged migrations in git history

**Review by ntruchsess — APPROVED** [review]

**Pull request merged**